contrib/devtools/symbol-check.py

```diff
@@ -142,26 +145,15 @@ def close(self):
 
 def read_symbols(executable, imports=True) -> List[Tuple[str, str, str]]:
```

Comment:
I don't know much about python type specifications but this is wrong now, I think it is `Tuple[List[Tuple[str, str]], int]`.